R/RcppExports.R

```diff
@@ -2,214 +2,214 @@
 # Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
 .agglomerative_clustering <- function(x, pairwise_conn, metric, n_neighbors, n_clusters) {
-    .Call('_cuda_ml_agglomerative_clustering', PACKAGE = 'cuda.ml', x, pairwise_conn, metric, n_neighbors, n_clusters)
+    .Call(`_cuda_ml_agglomerative_clustering`, x, pairwise_conn, metric, n_neighbors, n_clusters)
```

Comment:
Is there a special reason for not using the `PACKAGE='cuda.ml'` specifications?

Reply:
Nope. The changes were auto-generated, and I'm not sure why it switched from one form to another. Either form works for me though.